api/v1alpha3/conversion.go

```diff
@@ -117,6 +119,10 @@ func Convert_v1alpha3_OpenStackClusterSpec_To_v1beta1_OpenStackClusterSpec(in *O
 			Name: in.CloudsSecret.Name,
 		}
 	}
+	out.APIServerLoadBalancer = infrav1.APIServerLoadBalancer{
+		Enabled:         in.ManagedAPIServerLoadBalancer,
+		AdditionalPorts: *(*[]int)(unsafe.Pointer(&in.APIServerLoadBalancerAdditionalPorts)),
```

Comment:
What's the reason for using unsafe here? couldn't it be as easy as the following?:

```suggestion
		AdditionalPorts: in.APIServerLoadBalancerAdditionalPorts,
```